pysb/core.py

```diff
@@ -1271,8 +1271,16 @@ def __getnewargs__(self):
         return (self.name, self.value, False)
 
     def __init__(self, name, value=0.0, _export=True):
-        self.value = float(value)
+        self.value = value
         Component.__init__(self, name, _export)
+
+    @property
+    def value(self):
```

Comment:
What's this new getter/setter for? I'm guessing round-trip testing exposed a code path where the normalization of `.value` to a float was getting bypassed?

Reply:
Yes, in the earm_1_3.py model where `.value` is used with integers, but these values were getting converted to floats after round-tripped to JSON.